### ISSUE TITLE
File path is not read correctly from package.json

### ISSUE THREAD

**Issue by cuschk** [SamVerschueren/cache-conf#3]
The package name is `foo`. `package.json` looks like this:

``` json
{
  "name": "foo",
  "dependencies": {
    "cache-conf": "^0.2.0",
    "conf": "^0.11.2"
  }
}
```

Works with [`conf`](https://github.com/sindresorhus/conf) as expected:

``` js
const Conf = require('conf');
const configConf = new Conf();

console.log(configConf.path);
//=> /home/usch/.config/foo-nodejs/config.json
```

`cache-conf` looks like this:

``` js
const CacheConf = require('cache-conf');
const configCache = new CacheConf();

console.log(configCache.path);
//=> /home/usch/.config/cache-conf-nodejs/config.json
```

With `projectName` set manually, everything is fine:

``` js
const CacheConf = require('cache-conf');
const pkg = require('./package.json');
const configCacheName = new CacheConf({projectName: pkg.name});

console.log(configCacheName.path);
//=> /home/usch/.config/foo-nodejs/config.json
```

**Comment by SamVerschueren** [comment]
I noticed this as well, never came to it to create a new issue. Thanks for that, will probably fix this in the coming days.

**Comment by SamVerschueren** [comment]
Fixed in `0.3.0`
